src/cs/Ssh/Events/SshRequestEventArgs.cs

```diff
@@ -63,6 +64,16 @@ public class SshRequestEventArgs<T> where T : SshMessage
 	/// </remarks>
 	public Task<SshMessage>? ResponseTask { get; set; }
 
+	/// <summary>
+	/// Gets or sets an action to be invoked AFTER any response message has been sent.
+	/// (The continuation will be invoked even if a response was not requested.)
+	/// </summary>
+	/// <remarks>
+	/// This enables a request handler to ensure additional messages are sequenced after the
+	/// response message.
+	/// </remarks>
+	public Func<Task>? ResponseContinuation { get; set; }
```

Comment:
It could be `Func<SshMessage, Task>` to make it flexible and handle any result of `ResponseTask`.

Reply:
Updated. That requires awaiting the `ResponseTask` even if a response was not requested... but that's probably better anyway to not have an unawaited task.